plugin.json

```diff
@@ -0,0 +1,24 @@
+{
+    "id": "mattermost-plugin-suggestions",
+    "name": "Suggestions",
+    "description": "Suggest channels to users on request.",
+    "version": "0.1.0",
+    "min_server_version": "5.12.0",
+    "server": {
+        "executables": {
+            "linux-amd64": "server/dist/plugin-linux-amd64",
+            "darwin-amd64": "server/dist/plugin-darwin-amd64",
+            "windows-amd64": "server/dist/plugin-windows-amd64.exe"
+        }
+    },
+    "settings_schema": {
+        "header": "The Mattermost Suggestions Plugin recommends interesting channels to the user",
+        "footer": "",
+        "settings": [{
+            "key": "PreCalculationPeriod",
+            "display_name": "Precalculation period",
+            "type": "text",
+            "help_text": "Channel recommendations are precalculated periodically. One can choose the period based on [cron specifications](https://godoc.org/github.com/robfig/cron). Examples: \n @daily - Run once a day, midnight. \n @hourly - Run once an hour, beginning of hour. \n @every 1h30m - Run every hour and thirty minutes. \n The default is @weekly"
```

Comment:
I this you want to add a default value here: https://developers.mattermost.com/extend/plugins/manifest-reference/#settings_schema.settings.default